front/src/modules/views/hooks/useTableViews.ts

```diff
@@ -37,25 +50,48 @@ export const useTableViews = ({
   const [updateViewMutation] = useUpdateViewMutation();
   const [deleteViewsMutation] = useDeleteViewsMutation();
 
-  const createViews = useCallback(
-    (views: TableView[]) => {
-      if (!views.length) return;
-
-      return createViewsMutation({
-        variables: {
-          data: views.map((view) => ({
-            ...view,
-            objectId,
-            type: ViewType.Table,
-          })),
-        },
-        refetchQueries: [getOperationName(GET_VIEWS) ?? ''],
-      });
-    },
-    [createViewsMutation, objectId],
+  const { createViewFilters } = useViewFilters({ availableFilters });
+  const { createViewSorts } = useViewSorts({ availableSorts });
+
+  const createViews = useRecoilCallback(
```

Comment:
I'm not sure that useRecoilCallback is justified here too. the consequence of using useCallback would be re-renders when filters and sorts are updated which seems legit
Also, why are we updating multiple views with selectedFilters and selectedSorts? 
same question regarding createViewsMutation (vs createViewMutation)

Reply:
I've changed it back to `useCallback`.

In order to avoid using several callbacks (`handleViewCreate`, `handleViewUpdate`, `handleViewDelete`) + props drilling, only one callback (`handleViewsChange`) was created (following the same idea as the `onSortsUpdate` prop at the time). This callback computes the diff between the previous persisted state and the changed state, then creates/updates/deletes accordingly. As a matter of fact, only one view at a time will be created/updated/deleted in the use cases we have, but it could create/update/delete many views at once if needed.

`useTableViewFields`, `useViewFilters` and `useViewSorts` are built in the same way, but it probably makes more sense in those hooks as we can create/update/delete several view fields/filters/sorts before persisting, while for views, only one can be changed at a time.

If we want to do this differently, we could for instance pass in `handleViewsChange`'s params what type of operation we want to do (create, update or delete) and pass only one view.

Reply:
Ok! I still think that as we are updating one view at a time, it doesn't make sense to expose a hook API that could update many. Let's keep it as simple as it can be!